server/models/interfaces/Alerts.ts

```diff
@@ -27,6 +27,7 @@ export interface AlertItem {
   trigger_name: string;
   detector_id: string;
   state: string;
+  error_message: string | null;
```

Comment:
Any reason to use `string | null` instead of `error_message?: string`?

Reply:
yes null is returned when there's no error. This is alerts API response when there's no error:

```
{
    "ok": true,
    "response": {
        "alerts": [
            {
                "detector_id": "jVI5xI0BTNGkIsuMcKjz",
                "id": "78994df1-ade1-4db8-9a32-b4e5a16c4e73",
                "version": 1,
                "schema_version": 5,
                "trigger_id": "flI5xI0BTNGkIsuMYqiH",
                "trigger_name": "Trigger 1",
                "finding_ids": [
                    "125ff374-f6ec-4e14-8228-664910223086"
                ],
                "related_doc_ids": [
                    "1|smallidx"
                ],
                "state": "ACTIVE",
                "error_message": null,
                "alert_history": [],
                "severity": "1",
                "action_execution_results": [
                    {
                        "action_id": "",
                        "last_execution_time": 1708394174483,
                        "throttled_count": 0
                    }
                ],
                "start_time": "2024-02-20T01:56:14.468Z",
                "last_notification_time": "2024-02-20T01:56:14.468Z",
                "end_time": null,
                "acknowledged_time": null
            }
        ],
        "total_alerts": 1
    }
}
```